vss-processor/src/main/kotlin/org/eclipse/kuksa/vssprocessor/VssDefinitionProcessor.kt

```diff
@@ -137,11 +137,13 @@ class VssDefinitionProcessor(
         }
     }
 
-    companion object {
+    private companion object {
         private const val PACKAGE_NAME = "org.eclipse.kuksa.vss"
         private const val FILE_NAME_PROCESSOR_POSTFIX = "Processor"
-        private const val ASSETS_BUILD_DIRECTORY = "intermediates/assets/"
-        private const val BUILD_FOLDER_NAME = "build/"
+
+        private val fileSeparator = File.separator
```

Comment:
Due to how we resolve the buildFolder the approach was not working on windows before due to the difference in fileSeparators ("/" vs "\"). I changed it, however did not test it. So we should probably test if VssSpec generation also works on Windows devices!

FIrst I thought this would only be an issue for windows runners, but in fact it's an issue for each and every windows user of our lib - so we need to test and fix this ;)